cloudify_cli/commands/audit_log.py

```diff
@@ -0,0 +1,51 @@
+import click
+
+from ..cli import cfy, helptexts
+from ..table import print_data
+
+AUDITLOG_COLUMNS = ['ref_table', 'ref_id', 'operation', 'creator_name',
+                    'execution_id', 'created_at']
+
+
+@cfy.group(name='auditlogs')
+@cfy.assert_manager_active()
+def auditlogs():
+    """Manage the audit log"""
+    pass
+
+
+@auditlogs.command(name='list',
+                   short_help='List audit log entries')
+@click.option('-c', '--creator-name',
+              help=helptexts.AUDIT_CREATOR_NAME)
+@click.option('-e', '--execution-id',
+              help=helptexts.AUDIT_EXECUTION_ID)
+@cfy.options.sort_by()
+@cfy.options.descending
+@cfy.options.pagination_offset
+@cfy.options.pagination_size
+@cfy.options.common_options
+@cfy.pass_logger
+@cfy.pass_client()
+def get(creator_name,
+        execution_id,
+        sort_by,
+        descending,
+        pagination_offset,
+        pagination_size,
+        logger,
+        client
+        ):
```

Comment:
cmon if you put the closing paren on a separate line then a trailing comma on the last line is kinda a must have :P
```suggestion
        client,
        ):
```